redesign.user.js

```diff
@@ -2064,6 +2089,10 @@ const SETTINGS = [
     new BooleanSetting('general.speiseplan', false),
     new BooleanSetting('general.googlyEyes', true),
     new BooleanSetting('general.semesterzeiten', false),
+    new SelectSetting('general.language', 'auto', [
+        'auto',
+        ...Object.keys(TRANSLATIONS),
+    ]),
```

Comment:
put this to the top of general (immediately after `'general.updateNotification'`) please

Reply:
I think this looks a bit weird
![](https://github.com/jxn-30/better-moodle/assets/28537132/e54a4dbb-a306-467e-b705-14e48a5420da)

Reply:
That is true. would it maybe look better if the select was fullwidth?
I think it is important to have the language at top but it should also look good.